src/panels/lovelace/cards/hui-conditional-card.js

```diff
@@ -0,0 +1,79 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import computeCardSize from '../common/compute-card-size.js';
+import createCardElement from '../common/create-card-element.js';
+
+class HuiConditionalCard extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        .hidden {
+          display: none;
+        }
+      </style>
+      <div id="card"></div>
+    `;
+  }
+
+  static get properties() {
+    return {
+      hass: {
+        type: Object,
+        observer: '_hassChanged',
+      },
+      _config: Object
+    };
+  }
+
+  ready() {
+    super.ready();
+    if (this._config) this._buildConfig();
+  }
+
+  setConfig(config) {
+    if (!config || !config.card || !Array.isArray(config.conditions) ||
+        !config.conditions.every(c => c.entity && (c.state || c.state_not))) {
+      throw new Error('Error in card configuration.');
+    }
+
+    this._config = config;
+    if (this.$) this._buildConfig();
+  }
+
+  _buildConfig() {
+    const config = this._config;
+    const root = this.$.card;
+    while (root.lastChild) {
+      root.removeChild(root.lastChild);
+    }
+
+    const element = createCardElement(config.card);
+    element.hass = this.hass;
+    root.appendChild(element);
+  }
+
+  getCardSize() {
+    const el = this.$.card.lastChild;
+    return el && this._visible ? computeCardSize(el) : 0;
```

Comment:
The problem with this is that it becomes unpredictable how your UI will look. For it to look a certain way after a refresh,a state needs to be on now. I think that we should aim for cardSize to be static